certify-sunbird-insurance.properties

```diff
@@ -1,41 +1,43 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at https://mozilla.org/MPL/2.0/.
```

Comment:
Lets have the license text in all the config files

Reply:
added to all files